Automattic/pocket-casts-android
# Migrate podcast header bottom from data binding

## Description


Related to: #1908 

This PR migrates bottom header view on podcast details from data binding to view binding.

## Review hints

1. I've prepared this PR in a way that it's easy to review the change commit by commit, and I advise doing so.
2. I find it helpful to check "Hide white spaces" option. It makes much more easy to review some XML changes, as indentation in the file was changed. 

![image](https://github.com/Automattic/pocket-casts-android/assets/5845095/f778447f-2a75-4c80-81c6-e2bd1695b88c)

## Testing Instructions

Please smoke test the bottom header on episode details. Open/close the view. Verify if "read more" on description has max 3 lines and works as expected. Check different podcasts, especially those which lack some data like "next episode".


## Checklist
 
#### I have tested any UI changes...
